### ISSUE TITLE
TooltipHost has no way to turn off aria-describedby

### ISSUE THREAD

**Issue by petdun2519** [OfficeDev/office-ui-fabric-react#1858]
For some cases, like if you are trying to add a tooltip to text that is ellipsied by css, you want a visual only tooltip that will not affect screenreaders (because the screenreader can read all the text in the HTML already and doesn't care if some of it is hidden via css)

#### Priorities and help requested (not applicable if asking question):

Are you willing to submit a PR to fix? Yes I have one 

Requested priority: Low

Products/sites affected: All

**Comment by mdahamiwal** [comment]
@petdun2519, just for clarity the ToolTipHost component doesn't attaches aria-describedby by default. Take https://fabricreact.azurewebsites.net/fabric-react/master/#/examples/Tooltip for example, we decorate a button with TooltipHost and that is just a visual aid until we provide an explicit `aria-describedby` on the button and pass the same id to TooltipHost that links it to its tooltip text. 

Long story short, what you are asking in this issue is already flexible. Let me know if there is any confusion. :)

PS: As I mentioned in your PR, we should remove the `aria-describedby` that tooltip uses on itself. It provides no accessibility improvement. In fact, its confusing to see tooltiphost attaching that attribute.

**Comment by petdun2519** [comment]
@mdahamiwal wow... didn't realize that haha. That aria-describedby should be removed!!! I am going to close this PR for now
